README.rst

```diff
@@ -213,6 +213,60 @@ database connections at ``fork``.
 
 .. _celery-documentation:
 
+Backends and brokers
+====================
+
+*Note: This section is not comprehensive of backends and brokers.*
+
+Celery has the ability to communicate and store with many different backends (Result Stores) and brokers (Message Transports).
+
+Redis
+-----
+
+Redis can be both a backend and a broker.
+
+**As a Broker:** Redis works well for rapid transport of small messages. Large messages can congest the system.
+
+`See broker documentation for details`_
+
+.. _`See broker documentation for details`: https://docs.celeryproject.org/en/stable/getting-started/brokers/redis.html
+
+**As a Backend:** Redis is a super fast K/V store, making it very efficient for fetching the results of a task call. As with the design of Redis, you do have to consider the limit memory available to store your data, and how you handle data persistence. If result persistence is important, consider using another DB for your backend.
+
+RabbitMQ
+--------
+
+RabbitMQ is a broker.
```

Comment:
I am missing here RabbitMQ as a backend usage using rpc:// backend

Reply:
That goes outside my experience with Celery, I'll look into learning more about it though.